src/Models/IServiceHymnFormatContentInfo.ts

```diff
@@ -0,0 +1,52 @@
+﻿export interface ITextContent {
+    arabicText: string;
+    copticText: string;
+    englishText: string;
+}
+
+export interface IHazzatContent {
+    arabicHazzat: string;
+    copticHazzat: string;
+    englishHazzat: string;
+}
+
+export interface IVerticalHazzatContent {
+    arabicVerticalHazzat: string;
+    copticVerticalHazzat: string;
+    englishVerticalHazzat: string;
+}
+
+export interface IVideoContent {
```

Comment:
I'm assuming this is a link to the video and not the video buffer itself

Reply:
Correct.... Next change will be to fix up the content a bit to make it consumable though.